src/components/GetImageForm.js

```diff
@@ -48,6 +42,10 @@ export default class GetImageForm extends Component{
   }
 
 
+  componentDidMount() {
+    this.fetchRoverImage();
```

Comment:
Since we do want to call fetch after mounting the component, we can just call `this.fetchRoverImage`. We could have also done so like this:
```javascript
componentDidMount = this.fetchRoverImage
```

The above works because both are methods of no arguments. As such, you could have technically done `onClick={this.componentDidMount}`, but that doesn't read as clearly as either option above.